runhouse/utils.py

```diff
@@ -890,3 +891,32 @@ def get_gpu_usage(collected_gpus_info: dict, servlet_type: ServletType):
         gpu_usage["utilization_percent"] = gpu_utilization_percent
 
     return gpu_usage
+
+
+####################################################################################################
+# ray utils
+####################################################################################################
+def init_remote_cluster_servlet_actor(
+    current_ip: str,
+    runtime_env: Optional[Dict] = None,
+    servlet_name: str = "cluster_servlet",
+    cluster_config: Optional[dict] = None,
+):
+
+    from runhouse.servers.cluster_servlet import ClusterServlet
+
+    remote_actor = (
+        ray.remote(ClusterServlet)
+        .options(
+            name=servlet_name,
+            get_if_exists=True,
+            lifetime="detached",
+            namespace="runhouse",
+            max_concurrency=1000,
+            resources={f"node:{current_ip}": 0.001},
+            num_cpus=0,
+            runtime_env=runtime_env,
+        )
+        .remote(cluster_config=cluster_config, name=servlet_name)
```

Comment:
just wondering what's the desired outcome of passing in cluster_config and name here change?

Reply:
This relates to the discussion with Josh above (cc'ed you). For testing purposes, I wanted to create a mocked invalid cluster servlet that will be killed. To do so, I needed to create a cluster servlet with specific invalid cluster_config, and a specific name. The name is passed so `kill_actors` will kill only the invalid cluster servlet, and not other valid cluster servlets that are being used for other tests. 

It does not affect the main cluster_servlet creation: in this case the cluster_config will be `None` and the name will be `cluster_servlet`, as it was before.

Reply:
is there a reason `name=servlet` is included in both `options` and `remote`? also seeing the discussion above, I generally agree with not updating code for the sake of testing - could we leave it as it was, and just copy over the same lines of code for that test (or create a helper function inside test rather than in the core code if it's used more than once in test?)

Reply:
Yeah, makes sense. Added the helper function to the tests/utils.py. 

The name was passed to `options` to set the name of the ray actor instance. I also passed it to `remote` so we could save the name of the ClusterServelt as an attribute (`self.name`). By doing that, in a case the ClusterServlet should be killed, we'll kill a the specific clusterServlet, and no all actors in `runhouse` namespace. 
Meaning, instead of running:
`kill_actors(gracefully=False, namespace="runhouse")`
we could run 
`kill_actors(gracefully=False, name=self.name)`

I thought that passing the name to the `kill_actors` method is a safer way to kill the cluster servlet (both "in real time" and during tests). Having said that, so far it looks like running kill_actors with the `namespace=runhouse` instead of the `actor_name=self.name` does not causing tests breakage. so I assume that for now could leave the changes to the bare minimum, and pass the name only to options, as we previously did. (We should keep an eye that test won't brake in the future as well).